src/scripts/components/UploadListItem.js

```diff
@@ -23,7 +23,7 @@ import {
   videoProgress,
   videoDuration,
   stakedAmount
-} from '../operators/VideoOperators'
```

Comment:
This was a workaround because `flow` was not aware of the `operators/` directory. This is now fixed